builtin/providers/aws/data_source_aws_vpc_peering_connection.go

```diff
@@ -0,0 +1,121 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsVpcPeeringConnection() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsVpcPeeringConnectionRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": &schema.Schema{
```

Comment:
From Go 1.7 onwards type is derived, therefore there is no need to add `schema.Schema`.